code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -49,11 +49,7 @@
 	. = ..()
 	to_chat(C, "[info_text]")
 
-	C.real_name = "[pick(GLOB.nightmare_names)]"
-	C.name = C.real_name
-	if(C.mind)
-		C.mind.name = C.real_name
-	C.dna.real_name = C.real_name
+	C.fully_replace_character_name("[pick(GLOB.nightmare_names)]")

```

Comment:
since this is a roundstart race option, would this leave PDA's/records out of sync too?

Reply:
Uh, nightmares aren't roundstart ? Anyway this is why this should be moved out of here to where species is set on case by case basis.

Reply:
oh wait yeah, was thinking of shadow people